src/html/create_songs_html.py

```diff
@@ -0,0 +1,192 @@
+# Tworzy piosenki w html
+import os
+from lxml import etree
+import src.html.read_song_xml as rsx
+
+
+def _add_chunk(chunk, parent, position):
+    """class chunk -> html span chunk"""
+    span_chunk = etree.SubElement(parent, "span", attrib={"class": "chunk"})
+    akord = etree.SubElement(span_chunk, "span", attrib={"class": "akord"})
+    span_ch = etree.SubElement(akord, "span", attrib={"class": "ch"})
+    span_ch.text = chunk.chord
+    span_content = etree.SubElement(span_chunk, "span", attrib={"class": "content"})
+    if position == 0:
+        if chunk.content.startswith(' '):
+            span_content.text = chunk.content
+        else:
+            span_content.text = ' ' + chunk.content
+    else:
+        span_content.text = chunk.content
+
+
+def _add_lyric(row, parent):
+    """class lyric -> html span lyric"""
+    span_lyric = etree.SubElement(parent, "span", attrib={"class": "lyric"})
+    for i, chunk in enumerate(row.chunks):
+        _add_chunk(chunk, span_lyric, i)
+
+
+def _add_chords(row, parent, class_name):
+    """class chords -> html span ch"""
+    span_chords = etree.SubElement(parent, "span", attrib={"class": class_name})
+    for chunk in row.chunks:
+        if len(chunk.chord) > 0:
+            span_ch = etree.SubElement(span_chords, "span", attrib={"class": "ch"})
+            span_ch.text = chunk.chord
+
+
+def _add_bis(row, div_row):
+    if str(type(row.bis)) == "<class \'bool\'>" and row.bis is True:
+        span_bis = etree.SubElement(div_row, "span", attrib={"class": "bis_active"})
+        span_bis.text = '  '
+    elif str(type(row.bis)) == "<class \'int\'>":
+        span_bis = etree.SubElement(div_row, "span", attrib={"class": "bis_active"})
+        span_bis.text = 'x' + str(row.bis)
+    else:
+        span_unbis = etree.SubElement(div_row, "span", attrib={"class": "bis_inactive"})
+        span_unbis.text = '   '
+
+
+def _add_row(row, parent):
+    """class row -> html div row with content"""
+    if (row.new_chords == 0) or (row.new_chords is False):
+        chords_over = "over_false"
+    else:
+        chords_over = "over_true"
+    div_row = etree.SubElement(parent, "div", attrib={"class": "row " + chords_over})
+    div_row.text = u'\u200d'
```

Comment:
Please consider doing this in _add_lyric. 
It would provide a nicer xml formatting...

Reply:
i don't know what's going on and you don't remember too